assets/constants.ts

```diff
@@ -5208,10 +5225,20 @@ export default {
     "description": "Spanish",
     "relatedLanguages": ["es"],
     "direction": "ltr"
+  }, {
+    "id": "sw",
+    "description": "Swahili",
+    "relatedLanguages": ["sw"],
+    "direction": "ltr"
   }, {
     "id": "sv",
     "description": "Swedish",
-    "relatedLanguages": ["sw"],
+    "relatedLanguages": ["sv"],
```

Comment:
This is going to be complicated, like we have lanaguge_code stored in opportunity models and other entities, right? DO we need to update those as well?